data-model-solr/src/main/java/org/mousephenotype/cda/solr/web/dto/DataTableRow.java

```diff
@@ -15,13 +15,15 @@
  *******************************************************************************/
 package org.mousephenotype.cda.solr.web.dto;
 
-
+import org.apache.commons.lang.StringUtils;
+import org.apache.hadoop.mapred.gethistory_jsp;
 import org.apache.solr.client.solrj.SolrServerException;
 import org.mousephenotype.cda.enumerations.SexType;
 import org.mousephenotype.cda.enumerations.ZygosityType;
 import org.mousephenotype.cda.solr.service.dto.BasicBean;
 import org.mousephenotype.cda.solr.service.dto.ImpressBaseDTO;
 import org.mousephenotype.cda.solr.service.dto.MarkerBean;
+import org.neo4j.cypher.internal.compiler.v2_2.planner.logical.steps.allNodesLeafPlanner;
```

Comment:
Do we need this import? Actually, do we need neo4j reference in the pom for anything?